packages/cli/src/ceramic-daemon.ts

```diff
@@ -180,34 +179,61 @@ class CeramicDaemon {
+    streamsRouter.getAsync('/:streamid', this.state.bind(this))
+    pinsRouter.getAsync('/:streamid', this.listPinned.bind(this))
+    pinsRouter.getAsync('/', this.listPinned.bind(this))
+    nodeRouter.getAsync('/chains', this.getSupportedChains.bind(this))
+    nodeRouter.getAsync('/healthcheck', this.healthcheck.bind(this))
+    documentsRouter.getAsync('/:docid', this.stateOld.bind(this)) // Deprecated
+    recordsRouter.getAsync('/:streamid', this.commits.bind(this)) // Deprecated
 
     if (!gateway) {
-      app.postAsync(toApiPath('/streams'), this.createStreamFromGenesis.bind(this))
-      app.postAsync(toApiPath('/commits'), this.applyCommit.bind(this))
-      app.postAsync(toApiPath('/pins/:streamid'), this.pinStream.bind(this))
-      app.deleteAsync(toApiPath('/pins/:streamid'), this.unpinStream.bind(this))
+      streamsRouter.postAsync('/', this.createStreamFromGenesis.bind(this))
+      commitsRouter.postAsync('/', this.applyCommit.bind(this))
+      pinsRouter.postAsync('/:streamid', this.pinStream.bind(this))
+      pinsRouter.deleteAsync('/:streamid', this.unpinStream.bind(this))
 
-      app.postAsync(toApiPath('/documents'), this.createDocFromGenesis.bind(this)) // Deprecated
-      app.postAsync(toApiPath('/records'), this.applyCommit.bind(this)) // Deprecated
+      documentsRouter.postAsync('/', this.createDocFromGenesis.bind(this)) // Deprecated
+      recordsRouter.postAsync('/', this.applyCommit.bind(this)) // Deprecated
     } else {
-      app.postAsync(toApiPath('/streams'), this.createReadOnlyStreamFromGenesis.bind(this))
-      app.postAsync(toApiPath('/commits'),  this._notSupported.bind(this))
-      app.postAsync(toApiPath('/pins/:streamid'),  this._notSupported.bind(this))
-      app.deleteAsync(toApiPath('/pins/:streamid'),  this._notSupported.bind(this))
+      streamsRouter.postAsync('/', this.createReadOnlyStreamFromGenesis.bind(this))
+      commitsRouter.postAsync('/',  this._notSupported.bind(this))
+      pinsRouter.postAsync('/:streamid',  this._notSupported.bind(this))
+      pinsRouter.deleteAsync('/:streamid',  this._notSupported.bind(this))
 
-      app.postAsync(toApiPath('/documents'), this.createReadOnlyDocFromGenesis.bind(this)) // Deprecated
-      app.postAsync(toApiPath('/records'),  this._notSupported.bind(this)) // Deprecated
+      documentsRouter.postAsync('/', this.createReadOnlyDocFromGenesis.bind(this)) // Deprecated
+      recordsRouter.postAsync('/',  this._notSupported.bind(this)) // Deprecated
     }
+
+    commitsRouter.use(errorHandler(this.diagnosticsLogger))
+    documentsRouter.use(errorHandler(this.diagnosticsLogger))
+    multiqueriesRouter.use(errorHandler(this.diagnosticsLogger))
+    nodeRouter.use(errorHandler(this.diagnosticsLogger))
+    pinsRouter.use(errorHandler(this.diagnosticsLogger))
+    recordsRouter.use(errorHandler(this.diagnosticsLogger))
+    streamsRouter.use(errorHandler(this.diagnosticsLogger))
+
+    baseRouter.use('/commits', commitsRouter)
```

Comment:
Nitpick: Can we put this block up above the bindings to specific endpoints so it's easier while reading through this `registerAPIPaths` function to follow the flow and figure out what the various API endpoints are?